assets/constants.go

```diff
@@ -8,6 +8,8 @@ const (
 	BucketName = "flamingo-bot"
 	// StrikeTableName is the name of the table where strikes are persisted
 	StrikeTableName = "FlamingoStrikes"
+	// StrokeTableName is the name of the table where strokes are persisted
+	StrokeTableName = "FlamingoStrokes"
```

Comment:
smh

Reply:
@danlemmond is correct here. There is no reason for this to be its own table.

Reply:
strokes are not strikes insofar as i can see, are you sure you want to store strokes in a table named strikes (and I don't think extra tables cost more money, its just based on storage)

Reply:
It's closely related user data in a schemaless database. There's no reason to spread it across multiple tables.

Reply:
alright, should probably change the table name then (in the future)

Reply:
I literally cannot

Reply:
?

Reply:
https://stackoverflow.com/questions/21410851/how-to-rename-a-dynamodb-table

Reply:
.